lms/static/sass/base/_layouts.scss

```diff
@@ -81,12 +85,16 @@ body.is-in-course {
 
   // site footer
   .wrapper-footer {
-    margin-top: $baseline;
+    margin-top: ($baseline*2);
     padding-right: 2%;
     padding-left: 2%;
 
-    footer#footer-openedx { // TODO check edX footer when it launches
+    footer#footer-openedx { // shame selector to match existing
       min-width: auto;
     }
   }
+
+  footer#footer-edx-v3 { // shame selector to match existing
+    margin-top: ($baseline*2);
```

Comment:
Why is this rule not added to the nested ``footer#footer-edx-v3`` selector above?